processing/data_collection/gazette/spiders/pr_curitiba.py

```diff
@@ -0,0 +1,117 @@
+                '__EVENTARGUMENT': 'activeTabChanged:{}'.format(month),
+                'ctl00_cphMasterPrincipal_TabContalegacyDealPooliner1_ClientState': '{{"ActiveTabIndex":{},"TabState":[true,true,true,true,true,true,true,true,true,true,true,true]}}'.format(month)
+            },
+            meta={"month": month},
+            callback=self.parse_month
+        )
+
+    def parse_month(self, response):
+        page_count = len(response.css(".grid_Pager:nth-child(1) table td").extract())
+        month = response.meta["month"]
+        # The first page of pagination cannot be accessed by page number
+        yield scrapy.FormRequest.from_response(
+            response,
+            formdata={
+                '__EVENTTARGET': 'ctl00$cphMasterPrincipal$TabContainer1',
+                'ctl00_cphMasterPrincipal_TabContalegacyDealPooliner1_ClientState': '{{"ActiveTabIndex":{},"TabState":[true,true,true,true,true,true,true,true,true,true,true,true]}}'.format(month),
+                '__EVENTARGUMENT': 'activeTabChanged:{}'.format(month),
+            },
+            callback=self.parse_page,
+        )
+        for page_number in range(2, page_count + 1):
+            yield scrapy.FormRequest.from_response(
+                response,
+                formdata={
+                    '__EVENTARGUMENT': 'Page${}'.format(page_number),
+                    '__EVENTTARGET': 'ctl00$cphMasterPrincipal$gdvGrid2'
+                },
+                callback=self.parse_page,
+            )
+
+    def parse_page(self, response):
+        for idx, row in enumerate(response.css(".grid_Row")):
+            pdf_date = row.css("td:nth-child(2) span ::text").extract_first()
+            gazette_id = row.css("td:nth-child(3) a ::attr(data-teste)").extract_first()
+            parsed_date = parse(f'{pdf_date}', languages=['pt']).date()
+            if gazette_id == '0':
+                starting_offset = 3
+                yield scrapy.FormRequest.from_response(
+                    response,
+                    formdata={
+                        '__LASTFOCUS': '',
+                        '__EVENTTARGET': 'ctl00$cphMasterPrincipal$gdvGrid2$ctl{num:02d}$lnkVisualizar'.format(num=(idx+starting_offset)),
+                        '__EVENTARGUMENT': '',
+                        '__ASYNCPOST': 'true'
+                    },
+                    callback=self.scrap_not_extra_edition,
+                    meta={"parsed_date": parsed_date}
+                )
+            else:
+                yield Gazette(
+                    date=parsed_date,
+                    file_urls=["http://legisladocexterno.curitiba.pr.gov.br/DiarioSuplementoConsultaExterna_Download.aspx?id={}".format(gazette_id)],
+                    is_extra_edition=True,
+                    territory_id=self.TERRITORY_ID,
+                    power='executive_legislature',
+                    scraped_at=dt.datetime.utcnow()
+                )
+
+    def scrap_not_extra_edition(self, response):
```

Comment:
`parse_regular_edition` is a better name for this method (it is scrape, not scrap anyway).